gammapy/cube/tests/test_psf_map.py

```diff
@@ -48,8 +68,9 @@ def test_make_psf_map():
     assert psfmap.data.shape == (4, 50, 25, 25)
 
 
-def test_psfmap(tmpdir):
-    psf = fake_psf3d(0.15 * u.deg)
+def build_test_psfmap(size, shape="gauss"):
```

Comment:
So far in Gammapy we never use the word "build", always "make". Suggest to do that here also for consistency with the rest of the codebase.

Reply:
Done